G1ANT.Addon.UI/G1ANT.Addon.UI/Panels/UIControlsPanel.cs

```diff
@@ -18,6 +18,7 @@ public override void Initialize(IMainForm mainForm)
         {
             base.Initialize(mainForm);
             InitRootElement();
+            ControlType.Button.GetType();
```

Comment:
It will not be called when the window tree panel is closed, move it to XPathUIElementBuilder or UIElement static constructor